src/dtoannotations/Validator.java

```diff
@@ -0,0 +1,61 @@
+package dtoannotations;
+
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Validator {
+
+    public List<String> validate(Object object) throws IllegalAccessException {
+        List<String> list = new ArrayList<>();
+        Class<?> objectClass = object.getClass();
+        Field[] declaredFields = objectClass.getDeclaredFields();
+        for (Field declaredField : declaredFields) {
+            declaredField.setAccessible(true);
+            Object fieldValue = declaredField.get(object);
+            if (declaredField.isAnnotationPresent(Length.class) && fieldValue != null) {
```

Comment:
If you want to check **fieldValue != null** in every IF statement, so create a global IF statement.